ha/fault_tolerance/fault_tolerance_driver.py

```diff
@@ -60,25 +58,21 @@ def _get_consumer(self) -> MessageBusConsumer:
 
     def process_message(self, message: str):
         """Callback method for MessageConsumer"""
-        Log.debug(f'Received the message from message bus: {message}')
-        result = False
-        # EventAnalyzer class instantiation is expected.
-        # This is not required. This will be called from EventAnalyzer class.
-        # But as its not available yet and to check newly added filter functionality,
-        # temporarily added this
-        result = self._cluster_resource_filter.filter_event(message.decode('utf-8'))
-        Log.info(f'Alert required: {result}')
+        Log.info(f'Received the message from message bus: {message}')
+        try:
+            EventAnalyzer(message.decode('utf-8'))
+        except Exception as err:
+            Log.error(f'Failed to analyze the event: {err}')
+            return CONSUMER_STATUS.FAILED
```

Comment:
Confrim that this return status is true. If the message itself was corrupted we should not ask for the same message to be read again otherwise will be stuck in infinite loop

Reply:
Right. But as message is coming from message bus. We can be sure that message json structure is proper and not corrupted. So, means there can be some other issue and retry can solve that. I thought of this way.

Reply:
Yes, however this return value will cause message retry even if there is some exception received in system health path (which is not related to the incoming message) 
Let us handle it separately , I am closing this comment for now